apps/crn-server/test/utils/odata.test.ts

```diff
@@ -0,0 +1,67 @@
+import { buildEqFilterForWords, makeODataFilter } from '../../src/utils/odata';
+
+describe('makeODataFilter', () => {
+  test('converts the object to a format accepted by OData', () => {
+    expect(
+      makeODataFilter({
+        title: 'some title',
+        link: 'https://somelink.com',
+        documentType: ['a', 'b', 'c'],
+      }),
+    ).toStrictEqual([
+      { 'data/title/iv': 'some title' },
+      { 'data/link/iv': 'https://somelink.com' },
+      {
+        or: [
+          { 'data/documentType/iv': 'a' },
+          { 'data/documentType/iv': 'b' },
+          { 'data/documentType/iv': 'c' },
+        ],
+      },
+    ]);
+  });
+
+  describe('when the filter is not defined', () => {
+    test('returns null', () => {
+      expect(makeODataFilter()).toBeNull();
+      expect(makeODataFilter(undefined)).toBeNull();
+    });
+  });
+});
+
+describe('buildEqFilterForWords', () => {
+  test('builds the query correctly', () => {
+    expect(
+      buildEqFilterForWords('some field', ['a', 'b', 'c'], 'subField'),
```

Comment:
i have to say using a space in the field name caused a lot of confusion for me. Took me a while to understand it was adding the subfield at the end. Your fixtures should be somewhat reflective of a real life use case - you could even use an example of a team and a nested ID field.